lib/trento/activity_log.ex

```diff
@@ -52,10 +53,42 @@ defmodule Trento.ActivityLog do
     Repo.all(query)
   end
 
+  @spec clear_expired_logs() :: :ok | {:error, any()}
+  def clear_expired_logs do
+    with {:ok, retention_time} <- get_retention_time(),
```

Comment:
An alternative could be the following
```suggestion
    with {:ok, %{retention_time: retention_time}} <- get_settings(),
```
so we can remove the private `get_retention_time`